.github/workflows/ci.yml

```diff
@@ -80,134 +80,152 @@ jobs:
+        run: ./scripts/ci/ci_prepare_image_on_ci.sh
+      - name: "Tests"
         run: ./scripts/ci/ci_run_airflow_testing.sh
 
   tests-p36-sqlite:
-    name: "Tests [Sqlite][Py3.6]"
+    name: "[Sqlite][Py3.6]"
     runs-on: ubuntu-latest
     needs: [statics, statics-tests]
     env:
-      TRAVIS_JOB_NAME: "Tests [Sqlite][Py3.6]"
       BACKEND: sqlite
       PYTHON_VERSION: 3.6
+      CI_JOB_TYPE: "Tests"
     steps:
       - uses: actions/checkout@master
-      - name: "Tests [Sqlite][Py3.6]"
+      - name: "Build CI image"
+        run: ./scripts/ci/ci_prepare_image_on_ci.sh
+      - name: "Tests"
         run: ./scripts/ci/ci_run_airflow_testing.sh
 
   tests-p36-mysql-integrations:
-    name: "Tests [MySQL][Py3.6][integrations]"
+    name: "[MySQL5.7][Py3.6][int]"
     runs-on: ubuntu-latest
     needs: [statics, statics-tests]
     env:
-      TRAVIS_JOB_NAME: "Tests [MySQL][Py3.6][integrations]"
       BACKEND: sqlite
       PYTHON_VERSION: 3.6
       MYSQL_VERSION: 5.7
       ENABLED_INTEGRATIONS: "cassandra kerberos mongo openldap rabbitmq redis"
       RUN_INTEGRATION_TESTS: all
+      CI_JOB_TYPE: "Tests"
     steps:
       - uses: actions/checkout@master
-      - name: "Tests [MySQL][Py3.6][integrations]"
+      - name: "Build CI image"
+        run: ./scripts/ci/ci_prepare_image_on_ci.sh
+      - name: "Tests"
         run: ./scripts/ci/ci_run_airflow_testing.sh
 
   tests-p36-mysql-providers:
-    name: "Tests [MySQL5.7][Py3.7][providers][kerberos]"
+    name: "[MySQL5.7][Py3.7][prov][kerb]"
     runs-on: ubuntu-latest
     needs: [statics, statics-tests]
     env:
-      TRAVIS_JOB_NAME: "Tests [MySQL5.7][Py3.7][providers][kerberos]"
       BACKEND: mysql
       PYTHON_VERSION: 3.7
       MYSQL_VERSION: 5.7
       ENABLED_INTEGRATIONS: "kerberos"
+      CI_JOB_TYPE: "Tests"
     steps:
       - uses: actions/checkout@master
-      - name: "Tests [MySQL5.7][Py3.7][providers][kerberos]"
+      - name: "Build CI image"
+        run: ./scripts/ci/ci_prepare_image_on_ci.sh
+      - name: "Tests"
         run: ./scripts/ci/ci_run_airflow_testing.sh tests/providers
 
   tests-p37-mysql-kerberos:
-    name: "Tests [MySQL5.7][Py3.7][core][kerberos]"
+    name: "[MySQL5.7][Py3.7][core][kerb]"
     runs-on: ubuntu-latest
     needs: [statics, statics-tests]
     env:
-      TRAVIS_JOB_NAME: "Tests [MySQL5.7][Py3.7][core][kerberos]"
       BACKEND: mysql
       PYTHON_VERSION: 3.7
       MYSQL_VERSION: 5.7
       ENABLED_INTEGRATIONS: "kerberos"
+      CI_JOB_TYPE: "Tests"
     steps:
       - uses: actions/checkout@master
-      - name: "Tests [MySQL5.7][Py3.7][core][kerberos]"
+      - name: "Build CI image"
+        run: ./scripts/ci/ci_prepare_image_on_ci.sh
+      - name: "Tests"
         run: ./scripts/ci/ci_run_airflow_testing.sh --ignore=tests/providers
 
   requirements:
-    name: "Generate requirements"
+    name: "Generate reqs"
     runs-on: ubuntu-latest
     needs: [statics, statics-tests]
     strategy:
```

Comment:
```suggestion
    strategy:
      fail-fast: false
```

Reply:
I think actually until we can re-run individual jobs, fail-fast is better idea. We will have to re-run all of them anyway, so it's better if they all fail fast if one of them fails.

WDYT @turbaszek ?

Reply:
No, there might be other errors that we might miss and will have to wait for CI to run again to know about it

Reply:
With the curent rate of failures I am afraid we will have those failures more often on the perfectly ok  builds :). But sure. Let's try!